service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingService.kt

```diff
@@ -47,12 +53,37 @@ interface SøknadsbehandlingService {
 
     data class BeregnRequest(
         val behandlingId: UUID,
-        val fradrag: List<Fradrag>,
+        val fradrag: List<FradragRequest>,
         val begrunnelse: String?,
-    )
+    ) {
+        data class FradragRequest(
+            val periode: Periode?,
+            val type: Fradragstype,
+            val månedsbeløp: Double,
+            val utenlandskInntekt: UtenlandskInntekt?,
+            val tilhører: FradragTilhører,
+        )
+
+        object IkkeLovMedFradragUtenforPerioden
+
+        fun toFradrag(stønadsperiode: Stønadsperiode): Either<IkkeLovMedFradragUtenforPerioden, List<Fradrag>> =
+            fradrag.map {
+                if (it.periode != null && stønadsperiode.periode inneholder it.periode) {
```

Comment:
`!inneholder`

Reply:
Note to self: bli flinkere på å lage noen små tester for lignende klasser.

Reply:
Endret/la til et par tester på service nivå i hvert fall.